config/fcos/v1_6_exp/translate.go

```diff
@@ -367,3 +374,79 @@ func buildGrubConfig(gb Grub) string {
 	superUserCmd := fmt.Sprintf("set superusers=\"%s\"\n", strings.Join(allUsers, " "))
 	return "# Generated by Butane\n\n" + superUserCmd + strings.Join(cmds, "\n") + "\n"
 }
+
+func (c Config) handleSelinux(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
+	rendered := types.Config{}
+	ts := translate.NewTranslationSet("yaml", "json")
+	var r report.Report
+
+	for i, module := range c.Selinux.Module {
+		yamlPath := path.New("yaml", "selinux", "module", i)
+		if module.Name != "" && module.Content != "" {
+			rendered = processModule(rendered, module, options, ts, r, yamlPath)
+		} else {
+			r.AddOnWarn(yamlPath, common.ErrFieldInvalid)
+		}
+	}
+
+	if len(rendered.Storage.Files) != 0 {
+		newFilesystem := types.Filesystem{
+			Device: "/dev/disk/by-label/boot",
+			Format: util.StrToPtr("ext4"),
+			Path:   util.StrToPtr("/boot"),
+		}
+
+		rendered.Storage.Filesystems = append(rendered.Storage.Filesystems, newFilesystem)
+		ts.AddFromCommonSource(path.New("yaml", "selinux", "module", 0), path.New("json", "storage"), newFilesystem)
+
+	}
```

Comment:
This sugar shouldn't need to add filesystem things.

Reply:
Just curious to know how it works:
Why we don't need this? 
Should I remove all the filesystem part?

Thanks!

Reply:
In the GRUB case, we need to add a file in the boot filesystem (`user.cfg`) which is why we need Ignition to mount it so we can write to it. In our case, all the logic happens in the rootfs, so there's no need to edit the bootfs.

Reply:
Ah that makes sense; yeah we were using the GRUB case as our backing for this and incorrectly assumed the filesystem was a requirement.